pages/form.js

```diff
@@ -0,0 +1,162 @@
+
+      <form onSubmit={handleSubmit}>
+        <section>
+          <div>
+            <label>Titel</label>
+            <input
+              type="text"
+              name="title"
+              maxLength={30}
+              placeholder="Titel"
+              value={title}
+              onChange={handleTitleChange}
+              required
+            />
+          </div>
+          <div>
+            <label>Standorte</label>
+            <select name="locations" required>
+              <option value="">Bitte auswählen</option>
+              <option value="stadtpark">Stadtpark</option>
+              <option value="elbe">Elbe</option>
+              <option value="alster">Alster</option>
+            </select>
+          </div>
+          <div>
+            <label>Aktivitäten</label>
+            <select name="activities" required>
+              <option value="">Bitte auswählen</option>
+              <option value="gassigehen">Gassi gehen</option>
+              <option value="sport">Sport</option>
+              <option value="otherthings">Sonstiges</option>
+            </select>
+          </div>
+        </section>
+        <section>
+          <h4>Altersgruppe</h4>
+          <div>
+            <div>
+              <label htmlFor="welpen">Welpen</label>
+              <input
+                type="checkbox"
+                name="ageGroup"
+                id="welpen"
+                data-displayname="Welpen"
+              />
+            </div>
+            <div>
+              <label htmlFor="junghunde">Junghunde</label>
+              <input
+                type="checkbox"
+                name="ageGroup"
+                id="junghunde"
+                data-displayname="Junghunde"
+              />
+            </div>
+            <div>
+              <label htmlFor="erwachsen">Erwachsen</label>
+              <input
+                type="checkbox"
+                name="ageGroup"
+                id="erwachsen"
+                data-displayname="Erwachsen"
+              />
+            </div>
+            <div>
+              <label htmlFor="senior">Senior</label>
+              <input
+                type="checkbox"
+                name="ageGroup"
+                id="senior"
+                data-displayname="Senior"
+              />
+            </div>
+          </div>
+        </section>
+        <section>
+          <div>
+            <label htmlFor="description">Beschreibung</label>
+            <textarea
+              name="description"
+              maxLength={300}
+              rows="4"
+              cols="50"
+              placeholder="Text - Beschreibung"
+            />
+          </div>
+        </section>
+        <section>
+          <div>
+            <label>Kalender</label>
+            <input type="date" name="calendar" min={today} required />
+          </div>
+        </section>
+
+        <section>
+          <div>
+            <StyledButton>ADD</StyledButton>
```

Comment:
An other suggestion, how it is if you name this button: "Create Appointment" ? :)

Reply:
Yes, thats better ;)

Reply:
changed it ^^